kitsune/sumo/templatetags/jinja_helpers.py

```diff
@@ -456,6 +456,11 @@ def static(path):
         return ''
 
 
+@library.global_function
+def datetime_now(format):
```

Comment:
There's already a `datetimeformat` template function earlier in this file. You could rename this to just `now` and have it return the datetime, and then use the existing template function to format it.